style.css

```diff
@@ -0,0 +1,16 @@
+#navbar {
+  float: left;
+  width: 20%;
+}
+
```

Comment:
Please use indentation

Reply:
- [ ] Great and polite review. kindly add the reason why you think the requested change will be important or be of help

Reply:
If we are not considering appropriate indentation between CSS blocks an error may it happen. So that always we have to consider it like inside parentheses or at the beginning of the line